pages/profile/[username].tsx

```diff
@@ -19,7 +19,7 @@ interface PageProps {
     username?: string
 }
 
-const {publicRuntimeConfig} = getConfig()
+const { publicRuntimeConfig } = getConfig()
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The spacing around the equals sign has been adjusted for better readability and consistency with JavaScript best practices. This change improves the maintainability of the code.
```diff
- const {publicRuntimeConfig} = getConfig()
+ const { publicRuntimeConfig } = getConfig()
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->